src/OpenSage.Game/Logic/Object/Update/BattlePlanUpdate.cs

```diff
@@ -43,12 +43,12 @@ public void ChangeBattlePlan(BattlePlanType newPlan)
         GameObject.Owner.InitializeStrategyData(_validMemberKindOf, _invalidMemberKindOf);
     }
 
-    internal override void OnDie(BehaviorUpdateContext context, DeathType deathType, BitArray<ObjectStatus> status)
-    {
-        ClearActiveBattlePlan();
-        // todo: remove unit bonuses (but don't disable units)
-        base.OnDie(context, deathType, status);
-    }
+    //internal override void OnDie(BehaviorUpdateContext context, DeathType deathType, BitArray<ObjectStatus> status)
+    //{
+    //    ClearActiveBattlePlan();
+    //    // todo: remove unit bonuses (but don't disable units)
+    //    base.OnDie(context, deathType, status);
+    //}
```

Comment:
guessing this is just to be implemented later?

Reply:
It is - I've added a comment explaining why it's commented out.